elastic/integrations
# [Azure OpenAI] Add saved search to the Logs dashboard.

- Enhancement

- This PR adds the saved search to load the overview of logs in the logs dashboard.
- Rename metrics name in the overview dashboard.
- Removes the LENS which gets duplicated after adding saved search.
- Change the time range to last 48 hours in the logs dashboard.


## Checklist



## How to test this PR locally

Install the integration and verify the dashboard loads data properly in all the dashbaords.

## Screenshots

![azure_open_ai_logs png](https://github.com/elastic/integrations/assets/101238137/0aae49bb-1b99-4472-9245-960f4e0cfec8)
![azure_open_ai_overview](https://github.com/elastic/integrations/assets/101238137/c20ba630-7a29-4d3f-a459-a758c04fe78d)
